_posts/2018-07-17-its-a-lot-more-snappy.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "It's A Lot More Snappy"
+date: 2018-07-17
+categories: PWA SPA JavaScript
+author: Rick Foxcroft
+image: img/repeatable-serverless-applications/less_servers.png
+---
+
+**It's A Lot More Snappy**
+_‘A real world application of PWA technology’_
+
+Recently we undertook the journey of building a PWA(progressive web app) for a client here in Perth.  The app, is a maintenance app that allows maintenance technicians to raise issues with equipment in geologically  isolated areas. Due to this isolation, it was important that app could function in an offline mode and to do so, the app needed to store some data on the device it was running on.
```

Comment:
Add a Space between PWA and the open brackets. Also, it would be wise to explain what a PWA is. This is Perth.. it could be a jump.

Reply:
"it was important that app" should be "it was important that the app"

Reply:
Recently, Mechanical Rock undertook the journey of building a progressive web app (PWA) for one of our clients.

Reply:
The PWA is a maintenance app that allows technicians to raise issues with equipment in geologically isolated areas.

Reply:
...the app needed to store ~~~some~~~ data on the device it was running on.